tools/apigee-sackmesser/cmd/deploy/deploy.sh

```diff
@@ -157,8 +157,6 @@ elif [ "$apiversion" = "apigee" ]; then
         -Denv="$environment" \
         -Dhosturi="$baseuri" \
         -Dproxy.name="$bundle_name" \
-        -Dusername="$username" \
```

Comment:
looking at <https://docs.apigee.com/api-platform/system-administration/using-oauth2> it looks like this will break OPDK user. we should still provide the option of username and password for them

Reply:
discussed offline